meerk40t/core/svg_io.py

```diff
@@ -517,9 +514,22 @@ def parse(self, element, context_node, e_list):
             context_node = self.regmark
             e_list = self.regmark_list
         ident = element.id
+        # Let's see whether we can get the label from an inkscape save
+        # But don't get too excited yet, it needs a place to sit -> WIP
+        my_label = ""
+        try:
+            my_label = element.values.get("{http://www.inkscape.org/namespaces/inkscape}label")
```

Comment:
This value would still be in the element.values dictionary even outside of loading code. Though it would be best to check the cascade of different values for this label (EG. we wouldn't be saving an inkscape label element but some saved elements might have a label).